app/apps/views.js

```diff
@@ -148,3 +151,28 @@ exports.app_details = function (req, res) {
 
   });
 };
+
+
+exports.app_edit = [
+  ensureLoggedIn('/login'),
+  function(req, res, next) {
+    // This is making three requests cuncurrently to get the app details,
+    // the list of bucket and the list of users.
+    // If all of them succeed it then renders `apps/edit.html`
+    app_request = api.get_app(req.params.id);
+    buckets_request = api.list_buckets();
+    users_request = api.list_users();
```

Comment:
These need `var` or `const`

Reply:
Ops, well spotted :)

We could probably start to [use ES6 `let` in place of `var`](https://softwareengineering.stackexchange.com/q/274342) - `let` has block scope, `var` function scope, conclusion was "Adopt let and const. Stop using var".